collections/nemo_nlp/nemo_nlp/modules/classifiers.py

```diff
@@ -9,10 +9,14 @@
 from nemo.backends.pytorch.common import MultiLayerPerceptron
 from nemo.backends.pytorch.nm import TrainableNM, LossNM
 from nemo.core.neural_types import *
+from nemo_nlp.transformer.utils import gelu
 
 from ..transformer.utils import transformer_weights_init
 
 
+ACT2FN = {"gelu": gelu, "relu": nn.functional.relu}
```

Comment:
let's merge it for now, but in the future, we should find a better way to map name to activation